ArticleShapeExtractor/modules/ExportInDesignArticlesToFolder.mjs

```diff
@@ -106,16 +106,25 @@ class ExportInDesignArticlesToFolder {
                     }
                     articleShapeJson.textComponents.push(textComponent);
                 } else if (this.#inDesignArticleService.isValidArticleGraphicFrame(element.itemRef)) {
-                    pageItems.push(element.itemRef);
-                    articleShapeJson.imageComponents.push({
-                        "geometricBounds": this.#composeGeometricBounds(outerBounds.topLeftX, outerBounds.topLeftY, element.itemRef),
-                        "textWrapMode": this.#getTextWrapMode(element.itemRef)
-                    });
+                    const geometricBounds = this.#composeGeometricBounds(outerBounds.topLeftX, outerBounds.topLeftY, element.itemRef)
+                    if (geometricBounds.height > 10 && geometricBounds.width > 10) {
```

Comment:
I doubt if a hard 10 is really wanted; 
* An 8x8 could be some icon or decoration item?
* 12x2000 also looks like a line?

Should we use a proportional factor instead?
Something that indicates a pillar or bar?